packages/react-native/React/CoreModules/RCTDeviceInfo.mm

```diff
@@ -75,6 +75,10 @@ - (void)initialize
                                            selector:@selector(interfaceFrameDidChange)
                                                name:RCTWindowFrameDidChangeNotification
                                              object:nil];
+  [[NSNotificationCenter defaultCenter] addObserver:self
+                                           selector:@selector(interfaceFrameDidChange)
```

Comment:
@cipolleschi Should that really be calling `interfaceFrameDidChange` and not `interfaceOrientationDidChange` instead? I'm currently investigating an issue on 0.74.1, where orientation changes are not propagated correctly by `Dimensions` and/or `useWindowDimensions` and the culprit seems to be that nothing updates the dimensions properly on orientation change.

Reply:
Actually, after some more debugging - `[self _exportedDimensions]` is always returning the previous dimension prior the orientation change, so the portrait dimensions when switching to landscape and vice versa. Is this related when the `UIDeviceOrientationDidChangeNotification` event is triggered or related to when we gather the dimension data?

Reply:
Maybe related: https://snippet.zone/2022/11/